_docs/sysadmin/installation/version_notes/v22.06.00.md

```diff
@@ -0,0 +1,34 @@
+---
+title:  v22.06.00 > Course User/Role for Database
+category: System Administrator > Installation Version Notes
+---
+
+Release
+[v22.06.00](https://github.com/Submitty/Submitty/releases/v22.06.00)
+adds a new Postgres database user/role for course database access
+that is not a database superuser.
+
+Before installing this new version, you should specify the username
+and password for this new database user/role.  You do not need to
+create the user/role for the database, the migration will create the
+user/role if it does not exist.
+
+1.  As root, edit the file `/usr/local/submitty/config/database.json`
+
+    Add the following lines to the file:
+
+    ```
+    "database_course_user": "submitty_course_dbuser",
+    "database_course_password": "PASSWORD",
+    ```
+
+    Replacing `PASSWORD` with a secure password.  You may optionally
+    specify an alternate username for this database user/role.
+
+
+2.  Pull the latest code from the repostiroy, and run the installation
```

Comment:
```suggestion
2.  Pull the latest code from the repository, and run the installation
```

Reply:
Anyone know why the new codespell action didn't catch this?